src/black/nodes.py

```diff
@@ -25,14 +16,13 @@
 
 from mypy_extensions import mypyc_attr
 
-# lib2to3 fork
-from blib2to3.pytree import Node, Leaf, type_repr, NL
-from blib2to3 import pygram
-from blib2to3.pgen2 import token
-
 from black.cache import CACHE_DIR
 from black.strings import has_triple_quotes
+from blib2to3 import pygram
+from blib2to3.pgen2 import token
 
+# lib2to3 fork
```

Comment:
Did this comment cause the next line to be in it's own section? It seems like it would make more sense above all blib2to3 imports.

Note there are a couple of other occurrences of this comment in different files that may need looking at. They could possibly all just be removed though tbh.

Reply:
I agree, I've removed them.